data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+				goto "decision"
+
+			label "dueyu"
+			`	The Gegno leans closer to your height and displays a peculiar expression - possibly one of disapproval. You panic a little, and some of the others seem surprised that they got this close to you. They then speak in a low rumble, kind of like a whisper spoken aloud. "Eyuu idu keiou ghynem uf, 'Dueyu'." Getting back up, the Gegno heads back to the other two behind them.`
+
+			branch "done with the gegno"
+				or
+					or
+						"reputation: Gegno Vi" <= -100
+						"reputation: Gegno Scin" <= -50
+					and
+						"reputation: Gegno Vi" < 0
+						"reputation: Gegno Scin" < 0
+						has "zydee interference"
+					and
+						has "Full Vi Backlash: offered"
+						has "dead augen"
+					and
+						"gegno: encounters" >= 6
+						has "gegno: no proclamation"
+
+
+			label "decision"
+			action
+				"reputation: Gegno" = 1
+			`	For one last time, the trio beckons to the crowd followed by a long speech. An image of a space station appears on the screen, and a large discussion begins between the other Gegno, whom have formed back up with each other. Some arguments between various Gegno in the audience start happening, meanwhile, your Gegno guide bumps into you and starts to direct you away from the discussion, back to the cage lift with nobody else batting an eye.`
+			`	Several minutes later, you are escorted back to the base of the tower, which is now surrounded by a large group of Gegno civilians, all gawking at the two of you. You're led back to your ship with your head spinning slightly, unsure what to make of that whole encounter. As they did the first time you were with them on the one desert planet, the Gegno heads for your ship and hands you a small compass device, this time marking a different location on the map.`
+			`	At the very least, you can take this as an indication that they are not done with you yet, so you prepare to travel to your next destination.`
+				launch
+
+
+			label "done with the gegno"
+			`	For one last time, the trio beckons to the crowd followed by a long speech. Afterwards, the crowd goes silent. Your Gegno guide bumps into you and starts to direct you back to the cage lift with nobody else batting an eye. Several minutes later, you are escorted back to the base of the tower, still as barren as it was earlier. You're lead back to your ship with your head spinning slightly, unsure what to make of that whole encounter. The Gegno remains at a distance from your ship as you prepare to depart.`
+			`	Before you leave, you watch as they lift their staff weapon and flip it around. An explosive cloud of smoke follows, and a sizable crater appears just barely avoiding your ship on the ground. It seems that you're no longer welcome here, though, the fact that they intentionally missed such a large target might have other implications. Regardless, it could have been you on the other end of that blast at any point during this journey.`
```

Comment:
```suggestion
			`	Before you leave, you watch as they lift their staff weapon and flip it around. An explosive cloud of smoke follows, and a sizable crater appears right beside the <ship>. It seems that you're no longer welcome here, although the fact that they intentionally missed such a large target might have other implications. Regardless, it could have been you on the other end of that blast at any point during this journey.`
```